pkg/webhook/kubernetes/deployment.go

```diff
@@ -139,6 +139,7 @@ func WaitForDeploymentToStabilize(d *appsv1.Deployment, ip string) error {
 		if err != nil {
 			return false, nil
 		}
+		defer resp.Body.Close()
```

Comment:
were you running into an issue that this fixed? since it's unrelated to the rest of this change I'd like to see it in a separate commit, just so we can better track it.

Reply:
I didn't have an issue, no. But the doc says

> It is the caller's responsibility to close Body.

So I assumed it was simply forgotten here (...and all other calls to `http.Get(...)` do call close).

Reply:
mmm ok I guess I'm fine with it here :)